generators/app/templates/client/package.json

```diff
@@ -0,0 +1,66 @@
+{
+  "name": "<%= _.kebabCase(appName) %>",
+  "dependencies": {
+    "react-hot-loader": "^3.0.0-beta.3",
+    "style-loader": "^0.13.1",
+    "babel-core": "^6.0.20",
+    "babel-eslint": "^4.1.3",
+    "babel-loader": "^6.2.4",
+    "babel-plugin-transform-decorators-legacy": "^1.3.4",
+    "babel-polyfill": "^6.9.1",
+    "babel-preset-es2015": "^6.9.0",
+    "babel-preset-react": "^6.11.1",
+    "babel-preset-stage-0": "^6.5.0",
+    "connect-history-api-fallback": "^1.2.0",
+    "css-loader": "^0.23.1",
+    "express": "^4.14.0",
+    "extract-text-webpack-plugin": "^1.0.1",
+    "file-loader": "^0.9.0",
+    "flat": "^2.0.1",
+    "json-loader": "^0.5.4",
+    "lodash": "^4.15.0",
+    "node-sass": "^3.8.0",
+    "proxy-middleware": "^0.15.0",
+    "react": "^15.2.1",
+    "react-dom": "^15.2.1",
+    "react-redux": "^4.4.5",
+    "react-router": "^2.6.0",
+    "react-router-redux": "^4.0.5",
+    "redux": "^3.5.2",
+    "redux-saga": "^0.11.1",
+    "sass-loader": "^4.0.0",
+    "url-loader": "^0.5.7",
+    "webpack": "^1.13.1",
+    "webpack-merge": "^0.14.0",
+    "whatwg-fetch": "^1.0.0",
+    "yaml-loader": "^0.4.0",
+    "dotenv": "^2.0.0",
+    "uuid": "^2.0.2",
+    "raw-loader": "^0.5.1",
+    "classnames": "^2.2.5",
+    "immutable": "^3.8.1",
+    "postcss": "^5.1.2",
+    "postcss-cssnext": "^2.7.0",
+    "postcss-reporter": "^1.4.1",
+    "postcss-loader": "^0.11.1"
+  },
+  "devDependencies": {
+    "chai": "^3.5.0",
+    "enzyme": "^2.4.1",
+    "foreman": "^1.4.1",
+    "generator-prelude": "0.1.4",
+    "mocha": "^3.0.2",
+    "mocha-webpack": "^0.6.0",
+    "react-addons-test-utils": "^15.3.1",
+    "webpack": "^1.13.2",
+    "webpack-dev-server": "^1.12.1",
+    "webpack-node-externals": "^1.3.3",
+    "yo": "^1.8.4"
+  },
+  "scripts": {
+    "heroku-postbuild": "webpack --config config/production.js",
+    "start": "node server.js & webpack-dev-server --progress --colors --config config/webpack.development.js",
```

Comment:
I could swear I had commented this before, but this will put the `node server.js` on the background, so it's not a good idea. When you Ctrl C, it will only reach webpack-dev-server, which is the only process in the foreground. You would have to press Ctrl C, then execute `fg` then Ctrl C again to exit both.